internal/cli/main.go

```diff
@@ -127,10 +150,18 @@ Database drivers: `+strings.Join(database.List(), ", ")+"\n")
 		formatPtr := createFlagSet.String("format", defaultTimeFormat, `The Go time format string to use. If the string "unix" or "unixNano" is specified, then the seconds or nanoseconds since January 1, 1970 UTC respectively will be used. Caution, due to the behavior of time.Time.Format(), invalid format strings will not error`)
 		createFlagSet.BoolVar(&seq, "seq", seq, "Use sequential numbers instead of timestamps (default: false)")
 		createFlagSet.IntVar(&seqDigits, "digits", seqDigits, "The number of digits to use in sequences (default: 6)")
+		help := createFlagSet.Bool("help", false, "Print help information")
+
 		if err := createFlagSet.Parse(args); err != nil {
 			log.Println(err)
 		}
 
+		if *help {
+			log.Println(createUsage)
```

Comment:
Use `fmt.Fprintln(os.Stderr, ...)` for help messages so they're not modified when `-verbose` is specified.